bin/asinit

```diff
@@ -344,7 +344,7 @@ function ensurePackageJson() {
       pkg["scripts"] = scripts;
       updated = true;
     }
-    if (!scripts["test"]) {
+    if (!scripts["test"] || scripts["test"] == "echo \"Error: no test specified\" && exit 1") {
```

Comment:
How about move this string literal to top constant for easily search and modify in future?)